packages/xstate-wallet/src/chain.ts

```diff
@@ -240,6 +240,13 @@ export class ChainWatcher implements Chain {
   private get signer() {
     if (!this.ethereumIsEnabled) throw new Error('Ethereum not enabled');
 
+    if (window.ethereum.isFakeWeb3) {
+      return new Wallet(
+        '0xccb052837ccafb700e34c0e0cc0f3e5fbee8f078f3fe6b4e5950c7c8acaa7bce',
```

Comment:
Should this be configured via an ENV variable, with default value `'0xccb052837ccafb700e34c0e0cc0f3e5fbee8f078f3fe6b4e5950c7c8acaa7bce'`?

Reply:
Yeah I was going to add that in a separate PR. We currently use this hard-coded value as it's [the default](https://github.com/decentraland/dappeteer/blob/85a52146942613daa3a8e5a102f3bbe8c6f32f0f/src/index.ts#L64) dappeteer account.

Reply:
Actually it should be a secret because technically anyone on the internet can drain the ETH in this account.